run.sh

```diff
@@ -1,5 +1,5 @@
 # Run BETY
-#docker run --name betydb -d -p 5432:5432 terraref/bety-postgis
+docker run --name betydb -d -p 5432:5432 terraref/bety-postgis
```

Comment:
can you instead add the docker-compose file as well as the commands to load the initial database.

Reply:
please check the docker-compose file. not 100% sure it is right.